develop/tutorials/articles/200-integrating-with-front-end-frameworks/02-integrating-with-vue-js.markdown

```diff
@@ -0,0 +1,211 @@
+# Integrating with Vue JS [](id=integrating-with-vue-js)
```

Comment:
The same considerations regarding `.es.js` and discouraged `1.x` version of bundler as in the React tutorial apply here.